integreat_cms/api/v3/events.py

```diff
@@ -88,6 +93,9 @@ def transform_event_translation(event_translation, recurrence_date=None):
         else event_translation.available_languages_dict,
         "thumbnail": event.icon.url if event.icon else None,
         "location": transform_poi(event.location),
+        "location_url": settings.BASE_URL + poi_translation.get_absolute_url()
+        if event.location
+        else None,
```

Comment:
```suggestion
        "location_url": settings.BASE_URL + poi_translation.get_absolute_url()
        if poi_translation
        else None,
```
This would crash for an event translation if its location has not been translated into the language of the event translation and if fallback translations are disabled.